app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -24,6 +24,7 @@
         @include core-19;
         padding-top: $gutter-one-third;
         position: static;
+        width: 100%;
```

Comment:
Right, this one is weird. We shouldn't need it, but some very weird Chrome-specific rendering bug appears when you don't have it, for ONLY SOME strings of characters, when `direction: rtl` is flipped on. May as well post pictures:

### Before

![screen shot 2016-03-23 at 19 17 46](https://cloud.githubusercontent.com/assets/1650875/13997872/0a7f99c8-f12c-11e5-8dca-f93cbc00ad58.png)

### After

![screen shot 2016-03-23 at 19 17 54](https://cloud.githubusercontent.com/assets/1650875/13997884/174b5700-f12c-11e5-979c-4c5eb9237102.png)